torchdynamo/variables/torch.py

```diff
@@ -244,6 +246,20 @@ def fake_softmax(input):
 
         return variables.LambdaVariable(fake_softmax, **options)
 
+    def _call_cross_entropy_loss(self, tx, args, kwargs, options):
+        def fake_cel(input, target):
```

Comment:
cel?

Reply:
cross entropy loss, but I'll rename it, the characters don't come out of my paycheck ;)